docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -3,8 +3,293 @@ title: Excel custom functions tutorial
 description: In this tutorial, you’ll create an Excel add-in that contains a custom function that can perform calculations, request web data, or stream web data.
 ms.date: 12/21/2018
```

Comment:
I wasn't planning to update the date since my initial changes didn't substantively impact the content of the tutorial (from the user perspective). However, I will update it now that I'm revising content of the tutorial per your other feedback. Thanks!